.storybook/main.ts

```diff
@@ -0,0 +1,20 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  addons: [
+    '@storybook/addon-webpack5-compiler-swc',
+    '@storybook/addon-onboarding',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
+    '@storybook/addon-interactions',
+    '@storybook/addon-a11y',
+    'storybook-addon-intl'
+  ],
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {}
+  },
+  stories: ['../app/components/**/stories/**/*.mdx', '../app/components/**/stories/**/*.stories.@(js|jsx|mjs|ts|tsx)']
+};
+export default config;
```

Comment:
I think we need to also add support for lingui. Because we move from react intl to lingui

Reply:
https://github.com/wednesday-solutions/react-template/tree/master/.storybook

Refer the files from here

Reply:
Right, Thank you for pointing that out! I've now added support for lingui